webextension/scripts/overview.js

```diff
@@ -27,10 +26,9 @@ function getArchiveDetails (url) {
     .text(captures.toLocaleString())
     if (captures > 0) {
       $('#total_captures').show()
-      getThumbnail(url)
+      // getThumbnail(url)
```

Comment:
Are we not going to show the thumbnail in overview context ?

Reply:
I commented it out because it's not over a secure connection and remember reading about api calls should be over https or might not pass review. (could be wrong...)

Reply:
But you know I'm probably just being overly cautious and would like to see thumbnails shown, so I will restore it back and if it passes then great.